corehq/messaging/scheduling/tests/test_recipients.py

```diff
@@ -46,6 +46,94 @@
 from testapps.test_pillowtop.utils import process_pillow_changes
 
 
+class GetFilterValueTest(TestCase):
+    domain = 'get-filter-value-test'
+
+    @classmethod
+    def setUpClass(cls):
+        super(GetFilterValueTest, cls).setUpClass()
+        cls.domain_obj = create_domain(cls.domain)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.domain_obj.delete()
+        super(GetFilterValueTest, cls).tearDownClass()
+
+    def testNoBracesJustReturnValue(self):
+        value_or_property = "test value"
+        value = ScheduleInstance()._get_filter_value(value_or_property)
+        self.assertEqual(value, value_or_property)
+
+    def testBracesReturnProperty(self):
+        with create_test_case(self.domain, 'thing', 'thing A', case_properties={"property": "value"}) as case:
+            value_or_property = "{property}"
+            value = ScheduleInstance(case=case)._get_filter_value(value_or_property)
+            self.assertEqual(value, "value")
+
+    def testIgnoreSpacesBracesReturnProperty(self):
+        with create_test_case(self.domain, 'thing', 'thing A', case_properties={"property": "value"}) as case:
+            value_or_property = "{ property }"
+            value = ScheduleInstance(case=case)._get_filter_value(value_or_property)
+            self.assertEqual(value, "value")
+
+
+class PassesUserDataFilterTest(TestCase):
```

Comment:
Nice tests, really helpful for helping me understand the changes

Reply:
thanks